src/routes/docs/advanced/self-hosting/sites/+page.markdoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: article
+title: Sites
+description: Harness the full power of self-hosted sites with Appwrite. Explore site deployment, management, and integration in your self-hosted environment.
+---
+
+{% partial file="configure-github-app" %}
+
+# Configure Sites runtimes {% #sites %}
+
+Not all site runtimes are enabled by default. Enable the runtimes that you need and disable unused runtimes to save disk space on your server.
+To enable a runtime, add it to the `_APP_SITES_RUNTIMES` environment variable as a comma-separated list.
+
+The example below would enable Static, Node.js 22, and Flutter 3.29 runtimes.
+
+```bash
+_APP_SITES_RUNTIMES=static-1,node-22,flutter-3.29
+```
+
+You can also configure the maximum timeout that can be set on individual Appwrite Sites. The maximum configurable timeout can be increased by changing the `_APP_SITES_TIMEOUT` environment variable. This environment variable changes the configurable maximum but does not alter existing configurations of individual sites.
+
+{% partial file="update-variables.md" /%}
+
+# SSL certificates for Sites domains {% #ssl-certificates %}
+
+Before setting up SSL certificates, ensure you have configured your DNS settings properly. You'll need to create a CNAME record that points your wildcard domain (e.g. `*.appwrite.myapp.com`) to your Appwrite domain.
```

Comment:
@Meldiron I wanted to confirm which env variable is for used for Sites domains

Reply:
`_APP_DOMAIN_SITES`

Reply:
Yes, I just caught that on Discord, will add today